### DISCUSSION TITLE
filter directory name

### DISCUSSION THREAD

**Discussion by yukulele** [cboxdoerfer/fsearch#479] [Q&A] [answered]
Hi!

As a Nodejs dev, I have a lot of `node_modules` directories on my disk, I don't want Fsearch to index them.

I tryed some exclude filter like `*/node_modules`, `*/node_modules/`, `**/node_module/*` or simply `node_modules` and no one seems to work.

what's the best way to do it?

**Comment by cboxdoerfer** [comment, marked as answer]
Hi,

`node_modules` should do the trick and it does work on my system with the latest stable release (0.2.2). Are you using any other filters separated by semi-colons, if yes, can you post the whole filter you're using?

**Reply by yukulele** [reply]
Indeed, I also filter folders starting with a dot with `.*`.

`node_modules` and `.*` both work but `node_modules;.*` doesn't.

**Reply by yukulele** [reply]
@cboxdoerfer can you reproduce the bug with `node_modules;.*`?

**Reply by cboxdoerfer** [reply]
No, `node_modules;.*` works on my system and excludes both `node_modules` and any hidden files or folders. Which version are you using?

**Reply by yukulele** [reply]
After further checking, it turns out that it works perfectly!

Thanks for your help and sorry for the inconvenience.
